components/infobox/wikis/formula1/infobox_show_custom.lua

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
why is this inside this pr?

Reply:
This should be a separate PR, but what is the use case for this Infobox?